crates/biome_html_formatter/tests/specs/html/elements/inline/mixed-expressions-elements.astro.snap

```diff
@@ -0,0 +1,56 @@
+---
+source: crates/biome_formatter_test/src/snapshot_builder.rs
+info: elements/inline/mixed-expressions-elements.astro
+---
+# Input
+
+```astro
+<div>
+	<span>Name:</span> {fullName} <span>Age:</span> {age}
+</div>
+
+<p><strong>Total:</strong> {price} <em>(tax included)</em></p>
+
+```
+
+
+=============================
+
+# Outputs
+
+## Output 1
+
+-----
+Indent style: Tab
+Indent width: 2
+Line ending: LF
+Line width: 80
+Attribute Position: Auto
+Bracket same line: false
+Whitespace sensitivity: css
+Indent script and style: false
+Self close void elements: never
+-----
+
+```astro
+<div>
+	<span>Name:</span>
+	{fullName}
+	<span>Age:</span>
+	{age}
+</div>
+
+<p>
+	<strong>Total:</strong>
+	{price}
+	<em>(tax included)</em>
+</p>
```

Comment:
This is significantly different from prettier's output, and shows no difference from biome's current output.

https://biomejs.dev/playground/?code=PABkAGkAdgA%2BAAoACQA8AHMAcABhAG4APgBOAGEAbQBlADoAPAAvAHMAcABhAG4APgAgAHsAZgB1AGwAbABOAGEAbQBlAH0AIAA8AHMAcABhAG4APgBBAGcAZQA6ADwALwBzAHAAYQBuAD4AIAB7AGEAZwBlAH0ACgA8AC8AZABpAHYAPgAKAAoAPABwAD4APABzAHQAcgBvAG4AZwA%2BAFQAbwB0AGEAbAA6ADwALwBzAHQAcgBvAG4AZwA%2BACAAewBwAHIAaQBjAGUAfQAgADwAZQBtAD4AKAB0AGEAeAAgAGkAbgBjAGwAdQBkAGUAZAApADwALwBlAG0APgA8AC8AcAA%2BAA%3D%3D&language=svelte